schemas/in-network-rates/README.md

```diff
@@ -91,3 +91,7 @@ Negotiated rates for items and services can come from a variety of billing code
 | All Patient Diagnosis Related Groups | AP-DRG | |
 | All Patient Refined Diagnosis Related Groups | APR-DRG | [AHRQ documentation](https://www.hcup-us.ahrq.gov/db/nation/nis/APR-DRGsV20MethodologyOverviewandBibliography.pdf) |
 | Ambulatory Payment Classifications | APC | [APC background information](https://www.acep.org/administration/reimbursement/reimbursement-faqs/apc-ambulatory-payment-classifications-faq/#question0) |
+| Local Code Processing |LOCAL | |
```

Comment:
Still working on getting an example from the discussion (https://github.com/CMSgov/price-transparency-guide/discussions/9) on what this might look like for a negotiated rate -- in the meantime, let's remove this.

Reply:
I don't have any examples for HIPPS or EAPG.   One of our clients, thought LOCAL could be used for their unique pricing scenarios that were based on multiple factors, such as their behavioral health scenarios.